itest/rust/src/object_tests/object_test.rs

```diff
@@ -1076,3 +1076,59 @@ fn double_use_reference() {
     double_use.free();
     emitter.free();
 }
+
+// ----------------------------------------------------------------------------------------------------------------------------------------------
+
+#[derive(GodotClass)]
+#[class(init, base=Object)]
+struct MultipleImplBlocks {}
+
+#[godot_api]
+impl MultipleImplBlocks {
+    #[func]
+    fn foo(&self) -> String {
+        "result of foo".to_string()
+    }
+}
+
+#[godot_api(secondary)]
+impl MultipleImplBlocks {
+    #[func]
+    fn bar(&self) -> String {
+        "result of bar".to_string()
+    }
+}
+
+#[godot_api(secondary)]
+impl MultipleImplBlocks {
+    #[func]
+    fn baz(&self) -> String {
+        "result of baz".to_string()
+    }
+}
+
+/// Test that multiple inherent '#[godot_api]' impl blocks can be registered.
+/// https://github.com/godot-rust/gdext/pull/927
+#[itest]
+fn godot_api_multiple_impl_blocks_pull_927() {
```

Comment:
The link is enough, no need to include `_pull_927` in the test name :slightly_smiling_face: